client/me/purchases/payment/edit-card-details/index.jsx

```diff
@@ -70,10 +71,10 @@ class EditCardDetails extends PurchaseCardDetails {
 
 				<CreditCardForm
 					apiParams={ this.getApiParams() }
+					createPaygateToken={ createPaygateToken( 'card_update' ) }
```

Comment:
One problem with this change is that it creates a new bound function on every render call, which we have a [linting rule](https://github.com/Automattic/wp-calypso/blob/master/.eslintrc.js#L54) to prevent. This means that this branch of the render tree will be considered "dirty" and updated on the DOM every time a render occurs, which could negatively impact performance.

One solution would be to just set `this.createPaygateToken = createPaygateToken( 'card_update' );` on the class in the constructor.